src/hotspot/share/memory/metaspaceShared.cpp

```diff
@@ -122,7 +123,27 @@ char* MetaspaceShared::symbol_space_alloc(size_t num_bytes) {
   return _symbol_region.allocate(num_bytes);
 }
 
-size_t MetaspaceShared::reserved_space_alignment() { return os::vm_allocation_granularity(); }
+void MetaspaceShared::init_alignments() {
+  assert(_core_region_alignment == 0, "call this only once");
+  _core_region_alignment = (size_t)os::vm_allocation_granularity();
```

Comment:
The above assignment should be moved inside the `if (DumpSharedSpaces) { ...` block.